data/data-pipeline/data_pipeline/content/config/csv.yml

```diff
@@ -20,9 +20,21 @@ fields:
   - score_name: Total categories exceeded
     label: Total categories exceeded
     format: int64
-  - score_name: Definition M (communities)
+  - score_name: Definition N (communities)
     label: Identified as disadvantaged
     format: bool
+  - score_name: Definition N (communities) (including adjacency index)
```

Comment:
The idea here is we now have TWO distinct mechanisms through which a tract can become a DAC. Either it meets the criteria explicitly in our methodology, OR it is surrounded by DACs.